input/pagecontent/comparison.md

```diff
@@ -1,1206 +1,395 @@
 
-A resource may be conformant to multiple implementation guides. In addition to AU Core profiles, a FHIR resource may be conformant to profiles published in project implementation guides, other national implementation guides, or international implementation guides.
+### Comparison from AU Core to International Patient Access, International Patient Summary, and US Core
 
-The table below provides a high-level comparison, indicating if a resource conformant to a particular AU Core profile also satisfies (i.e. is conformant to) a profile from [International Patient Access](https://hl7.org/fhir/uv/ipa/STU1/), [International Patient Summary](https://hl7.org/fhir/uv/ips/STU1.1/), or [US Core](https://hl7.org/fhir/us/core/STU6.1/). See [Relationship to AUCDI and other IGs](https://build.fhir.org/ig/hl7au/au-fhir-core/branches/ft_blockvote-1/relationship.html#relationship-to-aucdi-and-other-igsl) for more information on the relationship of AU Core to these implementation guides.
+The table below provides a profile-only comparison from AU Core to other implementation guides. It indicates whether a resource compliant with a particular AU Core profile also satisfies (i.e., is compliant with) a profile from [International Patient Access](https://hl7.org/fhir/uv/ipa/STU1/), [International Patient Summary](https://hl7.org/fhir/uv/ips/STU1.1/), or [US Core](https://hl7.org/fhir/us/core/STU7/). See [Relationship to AUCDI and other IGs](https://build.fhir.org/ig/hl7au/au-fhir-core/relationship.html) for more information on the relationship of AU Core to these implementation guides.
 
-Conformance in reverse is not guaranteed, i.e. a resource conforming to [International Patient Access](https://hl7.org/fhir/uv/ipa/STU1/), [International Patient Summary](https://hl7.org/fhir/uv/ips/STU1.1/), or [US Core](https://hl7.org/fhir/us/core/STU6.1/) **MAY NOT** conform to AU Core.
+Compliance in the reverse direction is not guaranteed. This means a resource that is compliant with International Patient Access, International Patient Summary, or US Core **MAY NOT** be compliant with AU Core. In the future, we may provide reverse comparisons and CapabilityStatement comparison.
 
-<table border="1" cellspacing="0" cellpadding="0">
+Levels of compliance:
+
+- **Full compliance**: An AU Core compliant resource meets all the requirements of the compared IPA, IPS, or US Core profile. Flagged with a <img src="green_checkmark.svg.png" width="20"/>.
+- **Partial compliance**: An AU Core compliant resource meets all mandatory requirements but fails to meet some optional constraints of the compared IPA, IPS, or US Core profile. Flagged with an <img src="orange_checkmark.svg.png" width="20"/>.
```

Comment:
Partial compliance - the definition is not the point of this. This should be maybe (and have a think on how this can be better):
An AU Core compliant resource is compatible, but additional changes may be needed to meet all requirements